arithmoi-quadratic-sieve.prof

```diff
@@ -1,567 +1,529 @@
-	Fri Jun 12 21:49 2020 Time and Allocation Profiling Report  (Final)
+	Fri Jun 12 23:19 2020 Time and Allocation Profiling Report  (Final)
```

Comment:
Please do not commit `*.prof` files. You can add `*.prof` to `.gitignore` instead.